app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt

```diff
@@ -12,8 +12,8 @@ import javax.inject.Inject
 private const val KEY_CURRENT_EXPANDED_LIST_INDEX = "CURRENT_EXPANDED_LIST_INDEX"
 
 /** Fragment that contains subtopic list for play mode. */
```

Comment:
Update this comment to `/** Fragment that contains subtopic list for lessons mode. */`

Reply:
@praradbhgarg always make sure that you reply to these comments, some of the comment-replies can be like, what changes did you make, if you have any questions on this comment feel free to mention that, or simply write "DONE" if you had done the suggested changes.

Reply:
Ok. Will take care next time

Reply:
Done

Reply:
No issues, we already knew that this PR is easy, but the focus behind such PRs to make you aware of how oppia as a product is and how we normally communicate on PRs. It's completely fine.